hudi-sync/hudi-datahub-sync/src/main/java/org/apache/hudi/sync/datahub/DataHubSyncClient.java

```diff
@@ -97,6 +100,16 @@ public DataHubSyncClient(DataHubSyncConfig config, HoodieTableMetaClient metaCli
     this.databaseName = datasetIdentifier.getDatabaseName();
```

Comment:
Same as above, the names are already here which is inferred from the `HoodieDataHubDatasetIdentifier `, we can return them directly for the getters.

Reply:
sure.